### ISSUE TITLE
IE10 not overriding `flex-basis` when using `flex` shorthand

### ISSUE THREAD

**Issue by simonsmith** [philipwalton/flexbugs#92]
This pen demonstrates the issue - http://codepen.io/simonsmith/pen/qOKoRv#0

**Chrome** (expected)
![screen shot 2015-10-31 at 01 04 32](https://cloud.githubusercontent.com/assets/360703/10861034/954c446c-7f6b-11e5-9dc5-5be6103e1662.png)

**IE10** (actual)
![screen shot 2015-10-31 at 01 05 22](https://cloud.githubusercontent.com/assets/360703/10861044/a15294be-7f6b-11e5-836c-7be3f00a30e1.png)

The full width cells are there to demonstrate the rationale for wanting to set `flex: 0 0 100%;` initially and then override later for an alternate layout.

By setting `flex-basis: auto` on the left cell it shrinks to fit the content and adding `flex: 1 1 0% !important;` I would expect the right cell to fill the remaining space. It's this override that doesn't seem to apply in IE10:

![screen shot 2015-10-31 at 01 10 14](https://cloud.githubusercontent.com/assets/360703/10861067/33ea8a52-7f6c-11e5-8cf3-c9db803b68e2.png)

I would expect `-ms-flex-preferred-size` within `u-sizeFill` to also be important. Instead it is overridden by `Grid-cell` which has lower specificity.

### Fix

I've found that oddly, putting the `u-` rulesets below the `Grid` fixes the issue, but this isn't ideal.
Instead this seems to solve it more robustly:

``` css
.u-sizeFill {
  flex: 1 1 0% !important;
  flex-basis: 0% !important; /* fix */
}
```
I think this may be because in the prefixed output there is `-ms-flex-preferred-size: 0% !important;` present.

This feels like a valid bug (albeit odd) but if I'm using flex in an incorrect way please feel free to correct me! :)

**Comment by philipwalton** [comment]
Thanks, yeah I was able to reproduce this -- pretty weird bug. Can you let me know if https://github.com/philipwalton/flexbugs/pull/93 looks good, and I'll merge it in?

**Comment by philipwalton** [comment]
Closed via #93.
